### PR TITLE
Print operators info under modules and graphs

### PR DESCRIPTION
**Pull request by xiacijie**
Currently when we print `nn.ModuleBlock` or `nn.Graph`, only `INPUT`, `PARAMETER`, `BUFFER `and `OUTPUT `info will be shown. The operator info under `ModuleBlock `or Graph are not printed.

To solve this issue, the PR adds the functionality to show Operators info when we print `ModuleBlock `and `Graph `to help debugging.

Example：

```
(MODULE:model:ModuleMyLinear()): (
  (INPUT:_model_input.0.0_2:tensor(..., is_lazy='True', size=(1, 4), dtype=oneflow.float32))
  (PARAMETER:model.weight:tensor(..., size=(4, 3), dtype=oneflow.float32, requires_grad=True)): ()
  (PARAMETER:model.bias:tensor(..., size=(3,), dtype=oneflow.float32, requires_grad=True)): ()
  (MODULE:model.child:NullModule()): (
    (PARAMETER:model.child.x:tensor(..., size=(3, 3), dtype=oneflow.float32, requires_grad=True)): ()
    (PARAMETER:model.child.y:tensor(..., size=(3, 3), dtype=oneflow.float32, requires_grad=True)): ()
    (OPERATOR:model.child.x() -> (model.child.x/out))
    (OPERATOR:model.child.y() -> (model.child.y/out))
    (OPERATOR:model.child-matmul-0(model.child.x/out, model.child.y/out) -> (model.child-matmul-0/out_0))
    (OUTPUT:_model.child_output.0.0_2:tensor(..., is_lazy='True', size=(3, 3), dtype=oneflow.float32))
  )
  (OPERATOR:model.weight() -> (model.weight/out))
  (OPERATOR:model-matmul-1(_GraphMyLinear_0_input.0.0_2/out, model.weight/out) -> (model-matmul-1/out_0))
  (OPERATOR:model.bias() -> (model.bias/out))
  (OPERATOR:model-broadcast_add-2(model-matmul-1/out_0, model.bias/out) -> (model-broadcast_add-2/z_0))
  (OUTPUT:_model_output.0.0_2:tensor(..., is_lazy='True', size=(1, 3), dtype=oneflow.float32))
)
```

### CODE REVIEW

**Comment by xiacijie** [comment]
The original PR https://github.com/Oneflow-Inc/oneflow/pull/8112 is closed because that branch was renamed.

This PR is ready for review. @strint  @BBuf

File: python/oneflow/nn/graph/block.py [hunk 1]
```diff
@@ -679,13 +740,18 @@ def __getitem__(self, idx):
...
+        self,
+        prefix: str = "",
+        name: str = "",
+        origin: ParameterDict = None,
+        belonged_graph: weakref.ProxyTypes = None,
     ):
         super().__init__()
         self._name_prefix = prefix
         self._name = name
         self.set_origin(origin)
```
Comment (strint) [inline python/oneflow/nn/graph/block.py:752]: `self._belonged_graph = belonged_graph` should be moved before `set_origin` to get belonged_graph in sub_module

**Review by strint — APPROVED** [review]

File: python/oneflow/nn/graph/util.py [hunk 2]
```diff
@@ -15,8 +15,54 @@
...
+        output_sig_str = "..."
+
+        # only deal with UserOpConf and VariableOpConf for now
+        if op.HasField("user_conf"):
+            user_conf = op.user_conf
+            input_params = []
+            for param in user_conf.input_order:
+                x = user_conf.input[param].s
+                if len(x) > 1:  # param of multiple tensors
+                    input_params.append("[" + (", ").join(list(x)) + "]")
```
Comment (BBuf) [inline python/oneflow/nn/graph/util.py:42]: 这里需要考虑下那种多输入并且operand非常多的op吗？比如concat有10个输入，这样会不会让打印的repr比较丑，可以在单测试一试

Comment (xiacijie) [reply python/oneflow/nn/graph/util.py:44]:
```
(GRAPH:GraphMyLinear_0:GraphMyLinear): (
  (CONFIG:config:GraphConfig(training=False, ))
  (INPUT:_GraphMyLinear_0_input.1.0_key1:tensor(..., size=(3, 4), dtype=oneflow.float32))
  (MODULE:model:ModuleMyLinear()): (
    (INPUT:_model_input.0.0_2:tensor(..., is_lazy='True', size=(3, 4), dtype=oneflow.float32))
    (PARAMETER:model.bias:tensor(..., size=(3,), dtype=oneflow.float32, requires_grad=True)): ()
    (OPERATOR: model-FreeEagerTensor-1() -> (model-FreeEagerTensor-1/out))
    (OPERATOR: model-FreeEagerTensor-2() -> (model-FreeEagerTensor-2/out))
    (OPERATOR: model-FreeEagerTensor-3() -> (model-FreeEagerTensor-3/out))
    (OPERATOR: model-concat-0([model-FreeEagerTensor-1/out, model-FreeEagerTensor-2/out, model-FreeEagerTensor-3/out, model-FreeEagerTensor-1/out, model-FreeEagerTensor-2/out, model-FreeEagerTensor-3/out, model-FreeEagerTensor-1/out, model-FreeEagerTensor-2/out, model-FreeEagerTensor-3/out]) -> (model-concat-0/out_0))
    (OUTPUT:_model_output.0.0_2:tensor(..., is_lazy='True', size=(2, 54, 5, 3), dtype=oneflow.float32))
  )
  (OUTPUT:_GraphMyLinear_0_output.0.0_2:tensor(..., is_lazy='True', size=(2, 54, 5, 3), dtype=oneflow.float32))
)
```

Comment (xiacijie) [reply python/oneflow/nn/graph/util.py:44]: @BBuf 打印出来这个样子

Comment (BBuf) [reply python/oneflow/nn/graph/util.py:44]: 好的，我的意思就是说非常长的话可以考虑截断一下

**Review by BBuf — APPROVED** [review]

**Pull request merged**